cypress/support/commands/general.js

```diff
@@ -412,8 +412,6 @@ Cypress.Commands.add('waitForSaveIndicator', (expectIndicator = false) => {
   }
   cy.get('.indicator-pending').should('not.exist');
   cy.get('.indicator-saved').should('exist');
-  cy.get('.indicator-pending').should('not.exist');
-  cy.get('.indicator-saved').should('exist');
 });
```

Comment:
nice, I always wondered why do we need them twice.

Reply:
I was trying some stuff, but it seems my ideas didn't work.